cmd/crc/cmd/delete.go

```diff
@@ -35,11 +36,11 @@ func runDelete(arguments []string) {
 		deleteCache()
 	}
 
-	commandResult, err := machine.Delete(deleteConfig)
-	output.Out(commandResult.Success)
+	_, err := machine.Delete(deleteConfig)
 	if err != nil {
-		output.Out(err.Error())
+		errors.ExitWithMessage(1, err.Error())
 	}
+	logging.Info("CodeReady Containers instance deleted")
 }
```

Comment:
output.Out VS logging.Info is debatable, as the former will only output the string, while the latter wraps it in [] which looks odd:
```
CodeReady Containers instance deleted
```
VS

```
[CodeReady Containers instance deleted]
```